atomic_reactor/utils/cachi2.py

```diff
@@ -287,4 +292,89 @@ def clone_only(remote_source: Dict[str, Any]) -> bool:
     if pkg_managers is not None and len(pkg_managers) == 0:
         return True
 
+    # only git-submodule
+    if pkg_managers is not None and pkg_managers == ['git-submodule']:
+        return True
+
     return False
+
+
+def has_git_submodule_manager(remote_source: Dict[str, Any]) -> bool:
+    """Returns true when for specific remote source git-submodule manager is requested"""
+    pkg_managers = remote_source.get("pkg_managers") or []
+    return 'git-submodule' in pkg_managers
+
+
+def update_submodules(repopath: Path):
+    """Update submodules in the given repo"""
+    cmd = ["git", "submodule", "update", "--init", "--filter=blob:none"]
```

Comment:
What is the `--filter=blob:none` based on?

Reply:
I took it from Cachito to do exactly same behavior, I don't know

Reply:
Ah yup, found it https://github.com/containerbuildsystem/cachito/blob/2e4795a7687b70a468e4f686ef05988994af1a7b/cachito/workers/scm.py#L297